src/Symfony/Component/Form/Extension/Core/Type/CheckboxType.php

```diff
@@ -33,6 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         // doing so also calls setDataLocked(true).
         $builder->setData(isset($options['data']) ? $options['data'] : false);
         $builder->addViewTransformer(new BooleanToStringTransformer($options['value']));
+        $builder->setAttribute('false_is_empty', true);
```

Comment:
`_false_is_empty`. I would also add a comment to explain why we add it there, and that it not needed in 5.1 either.